docs/_includes/generated/web_configuration.html

```diff
@@ -52,6 +52,11 @@
             <td style="word-wrap: break-word;">(none)</td>
             <td>Path to the log file (may be in /log for standalone but under log directory when using YARN).</td>
         </tr>
+        <tr>
+            <td><h5>web.metric-fetcher.update-interval</h5></td>
```

Comment:
I'm not quite sure whether this is the right place, have you thought about making this a metric option instead? It seems questionable that the MQS and fetcher are configured differently even though they are so tightly connected.